AutoGVP/select-clinVar-submissions.R

```diff
@@ -80,7 +80,8 @@ submission_summary_df <- vroom(input_submission_file,
   dplyr::mutate(DateLastEvaluated = case_when(
     DateLastEvaluated == "-" ~ NA_character_,
     TRUE ~ DateLastEvaluated
-  ))
+  )) %>%
+  dplyr::filter(!ReviewStatus %in% c("no assertion provded", "no assertion criteria provided"))
```

Comment:
is there a typo here in provided?

Reply:
sorry about that, I've updated to correct the typo